### PR TITLE
Unit 3.4 Complete

### PR DESCRIPTION
**Pull request by MylesHammerschmidt**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -14,8 +17,14 @@ gulp.task('rel', 'Reload task', function () {
...
 
+gulp.task('sass', function () {
+  return gulp.src('./**/*.scss')
+    .pipe(sass().on('error', sass.logError))
+    .pipe(gulp.dest('./'));
+});
+
 gulp.task('watch', 'Watch task', function () {
-  gulp.watch(['./**'], ['rel']);
+  gulp.watch(['./**/*.html', './**/*.css', './**/*.scss'], ['rel', 'sass']);
```
Comment (rjpepmeyer) [inline gulpfile.js]: I would separate this into two lines, with the html and the css triggering the 'rel' task and the scss triggering the 'sass' task. Right now it gets triggered a couple times for sass changes, and you don't really need to run sass if it's triggered by an html change.

**Comment by rjpepmeyer** [comment]
Looks good to me 👍

**Comment by rjpepmeyer** [comment]
Actually checking out your nice new styles, I noticed something weird with font sizes. Turns out some of your h3 close tags are missing the /, resulting in increasingly larger headings.

**Comment by rjpepmeyer** [comment]
Another nitpick: where you have `<h1>John Smith</h1><h2>'s profile</h2>`, it shows up on two lines. Since it's part of the same heading, it'd be more semantically correct to have something like `<h2><strong>John Smith</strong>'s profile</h2>`, which would also fix it back to being on a single line. Just add a style like `h2 strong {font-size: 1.1em}`

**Comment by rginther** [comment]
This looks good to me. 👍

**Pull request merged**